SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -56,6 +215,9 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	FDirectoryPath GeneratedSchemaOutputFolder;
 
 public:
+	/** Launch configuration description. */
```

Comment:
Worth clarifying that this is the "Default launch config" used by the above tickbox, not the 'global' launch configuration that would affect other types of launch e.g. cloud?
(May also want to update the Generate default launch config button's comment, to say it's using the config specified in the Launch configuration description)